src/Peachpie.CodeAnalysis/Semantics/PhpOperationVisitor.cs

```diff
@@ -130,7 +130,11 @@ public virtual void VisitList(BoundListEx x)
 
         public virtual void VisitFieldRef(BoundFieldRef x)
         {
-            VisitTypeRef(x.ParentType);
+            if (x.ParentType != null)
```

Comment:
Well, I'll do it if you want, but I don't understand the reason. When inheriting a visitor, I was surprised that VisitTypeRef can be called with null.  The null check must be then done in the inherited visitor. Isn't it better to prevent possible null dereference and save an unnecessary method call by keeping the check in VisitFieldRef? Or am I missing something?